src/SFA.DAS.Apprenticeships.Web/Identity/Authentication/EmployerAccountPostAuthenticationClaimsHandler.cs

```diff
@@ -22,14 +22,14 @@ public EmployerAccountPostAuthenticationClaimsHandler(IEmployerAccountService ac
     public async Task<IEnumerable<Claim>> GetClaims(TokenValidatedContext tokenValidatedContext)
     {
         var claims = new List<Claim>();
-        var userId = tokenValidatedContext.Principal.Claims
+        var userId = tokenValidatedContext.Principal?.Claims
```

Comment:
We allow the collection of a null value here then throw on line 32 if its null. Would be better to throw here if Principle is null.